services/els-verify/src/main/resources/bootstrap-cloud.yaml

```
@@ -0,0 +1,30 @@
+---
+spring:
+  application:
+    name: cwa-data
+  cloud:
+    vault:
+      ssl:
+        trust-store: file:${SSL_VAULT_TRUSTSTORE_PATH}
+        trust-store-password: ${SSL_VAULT_TRUSTSTORE_PASSWORD}
+      enabled: true
+      generic:
+        enabled: false
+      kv:
+        enabled: true
+        backend: ${VAULT_BACKEND}
+        profile-separator: '/'
+        application-name: 'edus'
```

```suggestion
        application-name: 'els-verify'
```
??? or which VAULT repo are we using?